### PR TITLE
Delete files if desired

### PR DESCRIPTION
**Pull request by JasonTheKitten**
Don't if not

### CODE REVIEW

**Comment by nothjarnan** [comment]
I think this may conflict with the changes I'm making to the installer. I'll make a PR and we'll check.

**Comment by nothjarnan** [comment]
Take a look at #7, does it look like it'll conflict? @JasonTheKitten

File: install.lua [hunk 1]
```diff
@@ -12,6 +12,11 @@ local function formatFS()
       end
     end
     fs.delete("AxiomUI")
+    print("Delete extra files?")
+    if read() == "y" then
+      fs.delete("install.lua")
```
Comment (nothjarnan) [inline install.lua:17]: Wouldn't it be better to remove the current running program, as long as it isn't pastebin?

Comment (JasonTheKitten) [reply install.lua:17]: No. It gets written by gitget. So even if we did not download it, it will exist

Comment (nothjarnan) [reply install.lua:17]: You're right.

**Pull request merged**